bounties/MetaMask Snap for the XRP Ledger.md

```diff
@@ -1,34 +0,0 @@
----
```

Comment:
I'm confused why this PR is sayin git's deleting this because I can't find the file in `main` anyway... weird.

Reply:
I don't know tbh, it's related to the accidental proposal merge but I don't see the committed change on my end.